unittests/util/TestObject.h

```diff
@@ -135,3 +135,24 @@ CreateMetaDataForCancelOffer(
     uint32_t transactionIndex,
     int finalTakerGets,
     int finalTakerPays);
+
+/*
+ * Create a owner dir ledger object
```

Comment:
I'm not sure these comments help anything as they are the same as the function name.
If you wanted to make those into doxygen documentation then consider using `/** */` comment instead, otherwise i think they are safe to remove everywhere in this file, wdyt?